src/MSBuild.Conversion.Facts/MSBuildFacts.cs

```diff
@@ -200,6 +200,7 @@ public static class MSBuildFacts
             { "System.CodeDom", "4.6.0" },
             { "System.ComponentModel.Composition.Registration", "4.6.0" },
             { "System.ComponentModel.Composition", "4.6.0" },
+            { "System.ComponentModel.Annotations", "4.6.0" },
```

Comment:
Is this something that would need to be updated to 6.x? Or is this similar to our package maps?

Reply:
Good question. From a UA perspective, I don't think that's necessary because these are only used when updating the SDK style projects (still targeting NetFx) so just using package versions that work with .NET Framework is fine.

From a try-convert perspective, it may be nice to use versions that correspond to the new TFM in scenarios where users are changing TFM at the same time they update the csproj. That would be a much larger change, though, so I'd rather look at it separate from this one.

Reply:
Also, these versions are only used in fallback scenarios when we can't query NuGet for version information so priority is lower on updating this behavior.